build.gradle

```diff
@@ -75,7 +75,7 @@ project.ext {
 }
 
 wrapper {
-    gradleVersion = "6.3"
+    gradleVersion = "6.9"
```

Comment:
What's the purpose of this 
```
wrapper {
 ...
}
```

My live was before, without this on all my repos, perfect and I missed nothing.
For me it has the touch of noise.

Reply:
I don't really know, to be honest. I'm just leaving it there in case ByteHamster wants it there for some reason.

Reply:
It's a relict from 2014 when Gradle did not have the ability to upgrade using `./gradlew wrapper --gradle-version x.y` yet. I think it can safely be removed.

Reply:
I will do so, then.

Reply:
Only the Gradle change #5221